html-css-onboarding/user-profile.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/dist/output.css" rel="stylesheet" />
+    <title>User Profile</title>
+  </head>
+  <body>
+    <div>
+      <nav class="nav-bar">
+        <div class="logo">
+          <div class="oval"></div>
+          <a href="#" class="nav-item movy">Movy</a>
+        </div>
+      </nav>
+      <div class="edition-container">
+        <h2 class="font-medium leading-tight text-4xl content">Edit Profile</h2>
+        <div class="separator"></div>
+        <div class="edition">
+          <div class="image">
+            <img src="/html-css-onboarding/assets/images/profile.png" alt="" />
+          </div>
+          <form action="" class="edition-form">
+            <div class="first-line">
+              <input
+                type="text"
+                id="name"
+                class="p-2 w-full text-xl font-medium input leading-none"
+                required
+                placeholder="Your name"
+              />
+              <div class="flex self-center flex-1 items-end">
+                <input
+                  type="checkbox"
+                  class="input text-xl form-check-input appearance-none h-6 w-6 border border-gray-300 rounded-sm bg-white checked:bg-blue-600 checked:border-blue-600 focus:outline-none transition duration-200 mt-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
+                  id="kid"
+                />
+                <label
+                  class="form-check-label inline-block text-xl font-light"
+                  for="kid"
+                  >Kid?</label
+                >
+              </div>
+            </div>
+            <div class="mb-6 w-fit">
+              <label for="language" class="block mb-2 text-xl w-fit"
+                >Language:</label
+              >
+              <select id="language" class="p-2 text-xl input w-full" required>
```

Comment:
Chevrons for these seem to be too close to the input's border. Is there a way we can improve this?